frontend/src/components/material/ScheduleEntryLinks.vue

```diff
@@ -65,6 +66,19 @@ export default {
   },
   methods: {
     scheduleEntryRoute,
+    fullDescription(scheduleEntry) {
+      if (this.loading) return ''
+      return (
+        this.activity.title +
+        (scheduleEntry.number
+          ? `: ${scheduleEntry.number}`
+          : ` (${this.shortDescription(scheduleEntry)})`)
+      )
+    },
+    shortDescription(scheduleEntry) {
```

Comment:
If there are many ScheduleEntries for the Activity, the shortDescription takes more space:

![image](https://github.com/ecamp/ecamp3/assets/1506818/fdf4c8ea-72db-4461-88ae-84eb84247118)

Reply:
Yes, this is intended behaviour at this point. Do you have any proposals to improve this?

Reply:
No, for me this is ok.
Just wanted to make it transparent